godot-core/src/engine.rs

```diff
@@ -16,6 +16,40 @@ pub use crate::gen::central::global;
 pub use crate::gen::classes::*;
 pub use crate::gen::utilities;
 
+use self::packed_scene::GenEditState;
+
+/// Extension trait for convenience functions on `PackedScene`
+pub trait PackedSceneExt {
+    /// Instantiates the scene as type `T`, panicking if not found or bad type.
```

Comment:
```suggestion
    /// ⚠️ Instantiates the scene as type `T`, panicking if not found or bad type.
```

The emoji is for IDE inline docs, a convention we use for `Gd::cast()`, `Gd::from_instance_id()`, etc.